src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DictionaryGlobals.cs

```diff
@@ -76,6 +77,8 @@ static DictionaryGlobals()
                 RefLocalName = dictionary.Add(Globals.RefLocalName);
                 ArraySizeLocalName = dictionary.Add(Globals.ArraySizeLocalName);
                 EmptyString = dictionary.Add(String.Empty);
+                ISerializableFactoryTypeLocalName = dictionary.Add(Globals.ISerializableFactoryTypeLocalName);
```

Comment:
Opened https://github.com/dotnet/corefx/issues/13191 for tracking adding the missing entries back.